operator/config/manifests/bases/rhacs-operator.clusterserviceversion.yaml

```diff
@@ -752,6 +752,29 @@ spec:
           settings in this section will have no effect.
         displayName: Scanner Component
         path: scanner.scannerComponent
+      - displayName: Persistence
+        path: scannerV4.db.persistence
+      - description: Uses a Kubernetes persistent volume claim (PVC) to manage the
+          storage location of persistent data. Recommended for most users.
+        displayName: Persistent volume claim
+        path: scannerV4.db.persistence.persistentVolumeClaim
+      - description: The name of the PVC to manage persistent data. If no PVC with
+          the given name exists, it will be created. Defaults to "stackrox-db" if
```

Comment:
It should be, and it is because PersistentVolumeClaim is reused. The comment is true there's a kubebuilder annotation that is setting this `stackrox-db` as default claim name.

I wonder why the volume creation still works with the correct name if nothing is configured. Give me some time to check that.

Reply:
So apparently K8s does only apply the defaults if the parent object in the CR is not nil, since in my tests I did not set any values for persistence, the default name defined here was not applied.

In order to support correct default naming if persistenceVolumeClaim is not nil I had to introduce a dedicated ScannerV4Persistence/ScannerV4PersistentVolumeClaim struct.